### PR TITLE
Valentin/refactor-frontend-and-add-features-to-the-landingPage

### PR DESCRIPTION
**Pull request by ValentinNM**
UPDATED: Refactored the landing page, fixed styling template, cleaned and formated current lists
ADDEDL: MUI styling, Header, ViewMemeber and CreateTrainer components
This is a "messy" push, maybe I should have instead pushed individual features, but hoppe all ok this time??

### CODE REVIEW

File: src/Pages/ViewMemeber.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+import Button from "@mui/material/Button";
+
+export default function ViewMember(props) { 
+
+return( 
+    <section className="all-member-details-container two-column-grid-forms">
+        <ul>
+            {props.map((memeber, index)=> {
```
Comment (vherus) [inline src/Pages/ViewMemeber.jsx:9]: `props` is an object, did you test this?

File: src/components/CreateMembers.jsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
+import Button from "@mui/material/Button";
+
+export default function (props) {
+  return (
+    <section className="two-column-grid-forms">
+      <div className="available-classes">
+        <h2>Available Classes: </h2>
+        <ul>
+          {props.map((oneClass, index) => {
```
Comment (vherus) [inline src/components/CreateMembers.jsx:9]: Same thing here. `.map` is an Array method, but `props` is an object

File: src/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    transition: theme.transitions.create('width'),
+    width: '100%',
+    [theme.breakpoints.up('md')]: {
+      width: '100ch',
+    },
+  },
+}));
+
+export default function PrimarySearchAppBar() {
+  const [anchorEl, setAnchorEl] = React.useState(null);
```
Comment (vherus) [inline src/components/Header.jsx]: Let's keep some consistency here by importing useState at the top of the file

**Review by vherus — CHANGES_REQUESTED** [review]
This is a very big PR which makes it difficult to review, definitely try to keep them shorter next time like you mentioned

**Comment by ValentinNM** [comment]
> `props` is an object, did you test this? 

just setup the bare bones template.. component is not connected just yet
intention to push this is to merge so other can pull and implement the rest of the logic

**Comment by ValentinNM** [comment]
> Same thing here. `.map` is an Array method, but `props` is an object 

just setup the bare bones template.. component is not connected just yet
intention to push this is to merge so other can pull and implement the rest of the logic

**Pull request merged**